README.md

```diff
@@ -17,89 +17,62 @@
-| Append files          | Not started |
-| Rewrite files         | Not started |
-| Rewrite manifests     | Not started |
-| Overwrite files       | Not started |
-| Row level updates     | Not started |
-| Replace partitions    | Not started |
-| Snapshot management   | Not started |
-
-### Integrations
-
-We will add integrations with other rust based data systems, such as polars, datafusion, etc.
+# Apache Iceberg™ Rust
+
+
+
+Rust implementation of [Apache Iceberg™](https://iceberg.apache.org/).
+
+Working on [v0.3.0 Release Milestone](https://github.com/apache/iceberg-rust/milestone/2)
+
+## Components
+
+The Apache Iceberg Rust project is composed of the following components:
+
+| Name                   | Release                                                    | Docs                                                 |
+|------------------------|------------------------------------------------------------|------------------------------------------------------|
+| [iceberg]              | [![iceberg image]][iceberg link]                           | [![docs release]][iceberg release docs]              |
+| [iceberg-datafusion]   | -                                                          | -                                                    |
+| [iceberg-catalog-glue] | -                                                          | -                                                    |
+| [iceberg-catalog-hms]  | [![iceberg-catalog-hms image]][iceberg-catalog-hms link]   | [![docs release]][iceberg-catalog-hms release docs]  |
+| [iceberg-catalog-rest] | [![iceberg-catalog-rest image]][iceberg-catalog-rest link] | [![docs release]][iceberg-catalog-rest release docs] |
+
+[docs release]: https://img.shields.io/badge/docs-release-blue
+[iceberg]: crates/iceberg/README.md
+[iceberg image]: https://img.shields.io/crates/v/iceberg.svg
+[iceberg link]: https://crates.io/crates/iceberg
+[iceberg release docs]: https://docs.rs/iceberg
+
+[iceberg-datafusion]: crates/integrations/datafusion/README.md
+
+[iceberg-catalog-glue]: crates/catalog/glue/README.md
+
+[iceberg-catalog-hms]: crates/catalog/hms/README.md
+[iceberg-catalog-hms image]: https://img.shields.io/crates/v/iceberg-catalog-hms.svg
+[iceberg-catalog-hms link]: https://crates.io/crates/iceberg-catalog-hms
+[iceberg-catalog-hms release docs]: https://docs.rs/iceberg-catalog-hms
+
+[iceberg-catalog-rest]: crates/catalog/rest/README.md
+[iceberg-catalog-rest image]: https://img.shields.io/crates/v/iceberg-catalog-rest.svg
+[iceberg-catalog-rest link]: https://crates.io/crates/iceberg-catalog-rest
+[iceberg-catalog-rest release docs]: https://docs.rs/iceberg-catalog-rest
 
 ## Contribute
 
-Iceberg is an active open-source project. We are always open to people who want to use it or contribute to it. Here are some ways to go.
+Iceberg is an active open-source project. We are always open to people who want to use it or contribute to it. Here are
+some ways to go.
```

Comment:
Maybe good to include the ASF as well.
```suggestion
Apache Iceberg is an active open-source project, governed under the Apache Software Foundation (ASF). We are always open to people who want to use or contribute to it. Here are some ways to get involved.
```
This also includes Scott's suggestion.